README.md

```diff
@@ -145,91 +220,54 @@ kurtosis enclave inspect YOUR_ENCLAVE_ID_HERE
 Kurtosis will output everything it knows about your enclave, similar but not identical to the output below:
```

Comment:
We've actually already introduced `enclave inspect` at this point, so you can probably remove this part (which used to be the first point where `enclave inspect` was introduced, but is no longer)